barretenberg/cpp/src/barretenberg/translator_vm/translator_prover.cpp

```diff
@@ -134,8 +134,14 @@ void TranslatorProver::execute_relation_check_rounds()
         gate_challenges[idx] = transcript->template get_challenge<FF>("Sumcheck:gate_challenge_" + std::to_string(idx));
     }
 
+    const size_t log_subgroup_size = static_cast<size_t>(numeric::get_msb(Flavor::Curve::SUBGROUP_SIZE));
```

Comment:
I have absolutely no idea what any of this means, you'll need to ask someone from the crypto team to look at this

Reply:
This is just me pulling in Cody's PR with Mara and Luke's changes: https://github.com/AztecProtocol/aztec-packages/pull/13106

I also have no idea what it means, but we were trying to verify the ram blowup was solved